ansible/roles/pis/files/mrover.site

```diff
@@ -1,5 +1,8 @@
 [third_party]
 rapidjson=False
+nlohmann=False
+eigen=False
+kluge=False

```

Comment:
Please advise on whether this is necessary.

Reply:
see above quesiton